### ISSUE TITLE
Wrong german subdivisions

### ISSUE THREAD

**Issue by bib-cockpit** [openpotato/openholidaysapi#16]
Hello together,

by this request: https://openholidaysapi.org/Subdivisions?countryIsoCode=DE

I get a List with two wrong entries:

Allgemein bildende Schulen | MV-ABS
Berufliche Schulen | MV-BBS

This both entries are school types and no devision names.

**Comment by fstueber** [comment]
Please check the new groups endpoint: https://www.openholidaysapi.org/en/change-log-webservice/
For MV the school types are now under groups.

**Comment by BeyondVertical** [comment]
Good idea with the groups - shouldn't this also be added for France and the Zone A, B and C?

**Comment by fstueber** [comment]
That's the indeed the plan. This has already been done for the Netherlands and Belgium.
